### ISSUE TITLE
Failing to load templates located in app/assets/javascripts/templates

### ISSUE THREAD

**Issue by benams** [pitr/angular-rails-templates#95]
I try to set my routes and templates relations: 
```javascript
myApp.config(['$routeProvider',
  function($routeProvider) {
    $routeProvider.
      when('/', {
        templateUrl: 'index.html',
        controller: 'MainController'
      }).
      otherwise({
        redirectTo: '/'
      });
  }]);
```
But I get the following javascript error: Error: [$compile:tpload] Failed to load template: /index.html.

When I update my assets configuration:
```ruby
config/application.rb: config.assets.paths << Rails.root.join("app","assets","templates")
```
and set the full path: 'assets/index.html' it works as expected

Why it doesn't look for the templates in app/assets/javascripts/templates by default?

**Comment by vkeziah** [comment]
What is the version of your gem 'sass-rails' ?

**Comment by ilnurnasyrov** [comment]
i'm not sure, but it can be problem with sprockets. I move somebody.js file into app/assets/templates and insert
```js
//= requrie_tree ../templates
```
to application.js .
my somebody.js not compiled when i open page on the browser

**Comment by leonardoprg** [comment]
Yeah, it's a problem with new version of sprockets,

I'm using
```
gem 'sprockets', '2.12.3'
```
and it works

**Comment by vkeziah** [comment]
Yeah , I too fixed with this version

**Comment by albatrocity** [comment]
@leonardoprg thank you! This fixed my issue as well. Production environments were fine since the assets were precompiled, so I was perplexed on the dev environment.

**Comment by pitr** [comment]
Fixed in #96 Please see discussion in #93
